pkg/registry/rbac/rolebinding/policybased/storage.go

```diff
@@ -46,6 +47,14 @@ func (s *Storage) Create(ctx api.Context, obj runtime.Object) (runtime.Object, e
 		if s.superUser != "" && user.GetName() == s.superUser {
 			return s.StandardStorage.Create(ctx, obj)
 		}
+
+		// system:masters is special because the API server uses it for privileged loopback connections
```

Comment:
This is actually one of the options @erictune and I discussed around bootstrapping when we settled on the "super-user" option.

It sucks that this is going to stick around and be another user facing way to circumvent the escalation checks. Do you and @liggitt see this as being an eventual replacement for the super user flag? Or does this bootstrapping work end with the user not needing to ever utilize this? I guess I'm missing that context.